base/arraymath.jl

```diff
@@ -26,9 +26,100 @@ julia> A
 """
 conj!(A::AbstractArray{<:Number}) = (@inbounds broadcast!(conj, A, A); A)
 
-for f in (:-, :conj, :real, :imag)
-    @eval ($f)(A::AbstractArray) = broadcast_preserving_zero_d($f, A)
-end
+"""
+    conj(A::AbstractArray)
+
+Return an array containing the complex conjugate of each entry in array `A`.
+
+Equivalent to `conj.(A)`, except when `A` is a 0-dimensional result,
+in which case a 0-dimensional container is returned (rather than a scalar).
```

Comment:
Another difference is that these don't always copy:
```
julia> a = [1,2,3]; b = real(a); b[1] = 99; a
3-element Vector{Int64}:
 99
  2
  3
```

Reply:
Good catch. I've also noticed that the code is poorly organized, with methods scatters without any reason across abstractarraymath.jl and arraymath.jl. I'll regroup them.